package.json

```diff
@@ -25,16 +25,17 @@
   "devDependencies": {
     "chai": "^4.3.6",
     "chai-subset": "^1.6.0",
-    "eslint": "^8.18.0",
+    "eslint": "^8.19.0",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-plugin-import": "^2.26.0",
     "http-status-codes": "^2.2.0",
     "mocha": "^10.0.0",
-    "mochawesome": "^7.1.3"
+    "mochawesome": "^7.1.3",
+    "js-file-download": "^0.4.12"
   },
   "dependencies": {
     "axios": "^0.27.2",
-    "js-file-download": "^0.4.12",
+    "chai-json-schema": "^1.5.1",
     "md5": "^2.3.0",
     "md5-file": "^5.0.0"
```

Comment:
could you move these dependencies as dev dependency: `npm i -D chai-json-schema md5` and uninstall the md5-file since it is not needed